.github/workflows/cross.yml

```diff
@@ -28,7 +28,6 @@ jobs:
         # https://sourceforge.net/p/mingw-w64/wiki2/Exception%20Handling
         #
         image: [macos-latest, ubuntu-latest]
-        version: [stable, nightly]
```

Comment:
It was used only in `Add nightly toolchain with gnullvm target` step so `x86_64-pc-windows-gnu` actually ran twice on stable despite `nightly` word in the name of one of the jobs.